test/presubmit-tests.sh

```diff
@@ -57,9 +57,16 @@ function check_yaml_lint() {
     results_banner "YAML Lint" 0
 }
 
+function ko_resolve() {
+  header "Running `ko reslove`"
+  oci_layout_path=$(mktemp -d)
+  KO_DOCKER_REPO=example.com ko resolve --push=false --oci-layout-path=${oci_layout_path} -f config
```

Comment:
Why is `--oci-layout-path` needed? 🤔

Reply:
@ImJasonH because `ko` panics without 🙃 (and works with 🙃)

```
[…]
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x38 pc=0x14c34d5]

goroutine 140 [running]:
github.com/google/ko/pkg/resolve.ImageReferences.func1(0x0, 0x0)
        github.com/google/ko/pkg/resolve/resolve.go:66 +0x115
github.com/google/ko/vendor/golang.org/x/sync/errgroup.(*Group).Go.func1(0xc000518750, 0xc00063a9b0)
        github.com/google/ko/vendor/golang.org/x/sync/errgroup/errgroup.go:57 +0x59
created by github.com/google/ko/vendor/golang.org/x/sync/errgroup.(*Group).Go
        github.com/google/ko/vendor/golang.org/x/sync/errgroup/errgroup.go:54 +0x66
```

Reply:
I believe we fixed that in https://github.com/google/ko/pull/216 which should be in ko 0.6.1

Reply:
yeah, seems like I don't have `ko` 0.6.1 on my laptop 😅 (need to check what is packaged in test-runner too)

Reply:
https://github.com/tektoncd/plumbing/pull/660 for plumbing images.